sdk/synapse/Azure.Analytics.Synapse.AccessControl/tests/AccessControlClientLiveTests.cs

```diff
@@ -67,7 +67,7 @@ public static async ValueTask<Response> CreateResource(RoleAssignmentsClient ass
                     scope = "workspaces/" + testEnvironment.WorkspaceName
                 };
 
-                return await assignmentsClient.CreateRoleAssignmentAsync(roleAssignmentId, RequestContent.Create(roleAssignmentDetails));
+                return await assignmentsClient.CreateRoleAssignmentAsync(roleAssignmentId, RequestContent.Create(roleAssignmentDetails), ContentType.ApplicationJson);
```

Comment:
@pshao25, I see this test was updated -- are you still planning to update the CHANGELOG as well?

Reply:
Sorry I forgot yesterday. I was doing this just now. But I found updating Changelog needs to bump the version. Is this we expected?

Reply:
Any updates to the public API must be made to a new version of the library.